gatsby-node.js

```diff
@@ -219,6 +219,50 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     }
   });
 
+  const createEmbed = (node) => {
+    const {
+      frontmatter: { redirects },
+      fields: { fileRelativePath, slug },
+    } = node;
+
+    if (
+      fileRelativePath.includes('src/content/docs/release-notes') ||
```

Comment:
interesting. release note content does get surfaced in `nr1-help-xp`. buuuut let's be sure to exclude them from results via the docs library service before we implement the iframe in nr1-help-xp

Reply:
well if they get surfaced then I can remove this